contribute/style-guides/frontend.md

```diff
@@ -7,83 +7,267 @@ Generally we follow the Airbnb [React Style Guide](https://github.com/airbnb/jav
 - [Frontend Style Guide](#frontend-style-guide)
   - [Table of Contents](#table-of-contents)
   - [Basic rules](#basic-rules)
-  - [Organization](#organization)
-  - [Props](#props)
+  - [Naming conventions](#naming-conventions)
+  - [Files and directories naming conventions](#files-and-directories-naming-conventions)
+  - [Code organization](#code-organization)
+    - [Exports](#exports)
+  - [Comments](#comments)
+  - [React](#react)
+    - [Props](#props)
   - [State management](#state-management)
 
 ## Basic rules
 
 - Try to keep files small and focused.
 - Break large components up into sub-components.
+- Use spaces for indentation
 
-## Organization
+### Naming conventions
 
-- Components and types that needs to be used by external plugins needs to go into @grafana/ui
-- Components should get their own folder under features/xxx/components
-  - Sub components can live in that component folders, so small component do not need their own folder
-  - Place test next to their component file (same dir)
-  - Component sass should live in the same folder as component code
-- State logic & domain models should live in features/xxx/state
-- Containers (pages) can live in feature root features/xxx
-  - up for debate?
+#### Use `PascalCase` for:
 
-## Props
+1. Typescript class names
```

Comment:
Why are you using a numbered list here? Generally you only want to use numbered lists if the order of the items matters, like a procedure where you need to do certain steps in order. If order doesn't matter, then I recommend using a bulleted/unordered list instead.